db/migrate/20230613201507_create_users.rb

```diff
@@ -0,0 +1,19 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :string
+      t.string :Photo
+      t.string :string
```

Comment:
- [ ] From the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) there is no column in the `User` table as `string`. Kindly delete this column and ensure your migration follows the given ERD.